x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/layer/compatible_layer.tsx

```diff
@@ -0,0 +1,432 @@
+        } as unknown as estypes.MlJob,
+        undefined,
+        { max_model_memory_limit: '', effective_max_model_memory_limit: '' },
+        true
+      );
+
+      if (validationResults.contains('job_id_invalid')) {
+        setJobIdValid(
+          i18n.translate('xpack.ml.newJob.wizard.validateJob.jobNameAllowedCharactersDescription', {
+            defaultMessage:
+              'Job ID can contain lowercase alphanumeric (a-z and 0-9), hyphens or underscores; ' +
+              'must start and end with an alphanumeric character',
+          })
+        );
+      } else if (validationResults.contains('job_id_invalid_max_length')) {
+        setJobIdValid(
+          i18n.translate('xpack.ml.newJob.wizard.validateJob.jobIdInvalidMaxLengthErrorMessage', {
+            defaultMessage:
+              'Job ID must be no more than {maxLength, plural, one {# character} other {# characters}} long.',
+            values: {
+              maxLength: JOB_ID_MAX_LENGTH,
+            },
+          })
+        );
+      } else {
+        const resp = await ml.jobs.jobsExist([jobId]);
+        if (resp[jobId].exists) {
+          setJobIdValid(
+            i18n.translate('xpack.ml.newJob.wizard.validateJob.jobNameAlreadyExists', {
+              defaultMessage:
+                'Job ID already exists. A job ID cannot be the same as an existing job or group.',
+            })
+          );
+        }
+      }
+
+      if (validationResults.contains('bucket_span_invalid')) {
+        setBucketSpanValid(invalidTimeIntervalMessage(bucketSpan));
+      }
+      setState(STATE.DEFAULT);
+    },
+    500,
+    [jobId, bucketSpan]
+  );
+
+  return (
+    <>
+      {state !== STATE.SAVE_SUCCESS && state !== STATE.SAVING ? (
+        <>
+          <EuiFlexGroup gutterSize="s" data-test-subj="mlLensLayerCompatible">
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">
+                <EuiIcon type="checkInCircleFilled" color="success" />
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiText size="s">
+                {layer.jobWizardType === CREATED_BY_LABEL.MULTI_METRIC ? (
+                  <FormattedMessage
+                    id="xpack.ml.embeddables.lensLayerFlyout.createJobCalloutTitle.multiMetric"
+                    defaultMessage="This layer can be used to create a multi-metric job"
+                  />
+                ) : (
+                  <FormattedMessage
+                    id="xpack.ml.embeddables.lensLayerFlyout.createJobCalloutTitle.singleMetric"
+                    defaultMessage="This layer can be used to create a single metric job"
+                  />
+                )}
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="m" />
+          <EuiForm>
+            <EuiFormRow label="Job ID" error={jobIdValid} isInvalid={jobIdValid !== ''}>
```

Comment:
this `label` requires translation

Reply:
Updated in https://github.com/elastic/kibana/pull/136421/commits/ae35f0de4bed8502f2c0f97cacee58d0675c6f29